docs/detections/prebuilt-rules/rule-details/anomalous-process-for-a-windows-population.asciidoc

```diff
@@ -48,9 +48,9 @@ A newly installed program or one that runs rarely as part of a monthly or quarte
 
 ==== Investigation guide
 
-**Triage and analysis**
+*Triage and analysis*
 
-**Investigating an Unusual Windows Process**
+*Investigating an Unusual Windows Process*
 Detection alerts from this rule indicate the presence of a Windows process that is rare and unusual for all of the Windows hosts for which Winlogbeat data is available. Here are some possible avenues of investigation:
 - Consider the user as identified by the username field. Is this program part of an expected workflow for the user who ran this program on this host?
```

Comment:
Some of these don't transfer great to the asciidocs because they are actually written in MD destined for the security app in Kibana. In the shortly forthcoming revision of these docs, I think all of these investigation guides should be code escaped

Reply:
Oh, I had no idea some of these were written in Markdown, but that makes sense now.